examples/credential-registry/src/lib.rs

```diff
@@ -548,21 +538,21 @@ impl Deserial for CredentialEvent {
 pub struct InitParams {
     /// The issuer's metadata.
     issuer_metadata: MetadataUrl,
-    /// An address of the credential storage contract.
-    storage_address: ContractAddress,
     /// Credential schemas available right after initialization.
     #[concordium(size_length = 1)]
     schemas:         Vec<(CredentialType, SchemaRef)>,
     /// The issuer for the registry. If `None`, the `init_origin` is used as
     /// `issuer`.
-    issuer:          Option<AccountAddress>,
+    issuer_account:  Option<AccountAddress>,
+    /// The issuer's public key.
+    issuer_key:      PublicKeyEd25519,
     /// Revocation keys available right after initialization.
     #[concordium(size_length = 1)]
     revocation_keys: Vec<PublicKeyEd25519>,
 }
 
```

Comment:
We can remove
```
/// Mapping errors related to contract invocations to ContractError.
impl<T> From<CallContractError<T>> for ContractError {
    fn from(_cce: CallContractError<T>) -> Self { Self::ContractInvocationError }
}

```

and the `ContractInvocationError` in the CustomContractError.

Reply:
Good point. Removed now.